From bf0cbee86cb87d4bca45eafe889ae9c4b5146428 Mon Sep 17 00:00:00 2001
From: Joerg Bornemann <joerg.bornemann@qt.io>
Date: Mon, 24 Jun 2019 17:49:55 +0200
Subject: [PATCH 2/2] Do not ignore exit codes when installing meta files

Since commit 20e9422e we don't ignore exit codes when installing files
anymore. This patch does the same for meta file installation. We
really should be notified properly if something goes wrong here.

Task-number: QTBUG-18870
Change-Id: Ib6a20293380f400379b10ec767bf38dc74d5beeb
Reviewed-by: Oliver Wolff <oliver.wolff@qt.io>
Reviewed-by: Christian Kandeler <christian.kandeler@qt.io>
---
 qmake/generators/makefile.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/qmake/generators/makefile.cpp b/qmake/generators/makefile.cpp
index 57d3bb01b1..ab73fdc6d0 100644
--- a/qmake/generators/makefile.cpp
+++ b/qmake/generators/makefile.cpp
@@ -3422,7 +3422,7 @@ QString MakefileGenerator::installMetaFile(const ProKey &replace_rule, const QSt
     QString ret;
     if (project->isEmpty(replace_rule)
         || project->isActiveConfig("no_sed_meta_install")) {
-        ret += "-$(INSTALL_FILE) " + escapeFilePath(src) + ' ' + escapeFilePath(dst);
+        ret += "$(INSTALL_FILE) " + escapeFilePath(src) + ' ' + escapeFilePath(dst);
     } else {
         QString sedargs;
         const ProStringList &replace_rules = project->values(replace_rule);
@@ -3437,9 +3437,9 @@ QString MakefileGenerator::installMetaFile(const ProKey &replace_rule, const QSt
             }
         }
         if (sedargs.isEmpty()) {
-            ret += "-$(INSTALL_FILE) " + escapeFilePath(src) + ' ' + escapeFilePath(dst);
+            ret += "$(INSTALL_FILE) " + escapeFilePath(src) + ' ' + escapeFilePath(dst);
         } else {
-            ret += "-$(SED) " + sedargs + ' ' + escapeFilePath(src) + " > " + escapeFilePath(dst);
+            ret += "$(SED) " + sedargs + ' ' + escapeFilePath(src) + " > " + escapeFilePath(dst);
         }
     }
     return ret;
-- 
2.17.1

